src/functions/util/concatTextSplit.js

```diff
@@ -4,9 +4,9 @@ module.exports = async d => {
     const err = d.inside(inside);
     if (err) return d.error(err);
 
-    let [...text] = inside.splits;
+    let [...text, name = 'main'] = inside.splits;
     text = text.map(x => x.addBrackets());
```

Comment:
Same here, even if it used this method before, please update it to <data>.inside.splits